docs/spec/README.md

```diff
@@ -0,0 +1,161 @@
+# K8TRE Decisions
+
+> [!NOTE]  
+> The distinction between "the K8TRE Specification", "a K8TRE", and "the K8TRE Reference Implementation".
+> The K8TRE Specification is a SATRE-conformant Specification for how a K8S TRE should be built. *A* K8TRE is an implementation of a K8TRE Specification-compliant TRE. The K8TRE Reference Implementation is the official K8TRE project's implementation of a K8TRE Specification-compliant TRE.
+
+## GitOps using ArgoCD
```

Comment:
I don't think this file should be redundant with the associated files - i.e. let's remove the individual statements from this file. Maybe include an ToC for this folder though.